bfsbdump

```diff
@@ -53,6 +53,15 @@ def GetPlatformName():
 def CountSetBits(n):
     return bin(n).count("1")
 
+def GetEcid():
+    str = os.popen('xxd -p /sys/firmware/acpi/tables/SSDT1 2>/dev/null | tr -d "\\n"').readline()
+    pos1 = str.find('656369643031000e')
+    pos2 = str.find('656369643233000e')
```

Comment:
Question: what are those strings?
656369643031000e
656369643233000e

Reply:
This is the keyword in ACPI table: string "ecid01" followed by the first 4 bytes and string "ecid23" followed by the 2nd 4 bytes.

Reply:
318           Package () { "ecid01", 0x0102030405060708 },
319           Package () { "ecid23", 0x0BF3000101020304 }

Reply:
Ack